api/mesh/v1alpha1/gateway_route.proto

```diff
@@ -42,11 +42,10 @@ message MeshGatewayRoute {
     message Rule {
       // repeated Match matches = 1;
 
-      repeated Backend backends = 2
-          [ (validate.rules).repeated .min_items = 1 ];
+      repeated Backend backends = 2 [ (validate.rules).repeated.min_items = 1 ];
```

Comment:
these changes to proto files are result of updating our prehistoric version of `clang-format`